src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,102 @@
+import { NavLink } from 'react-router-dom'
+
+import 'css.gg/icons/css/play-button-r.css'
+import 'css.gg/icons/css/film.css'
+
+import Button from '../Button/Button'
+import styled from 'styled-components/macro'
+
+export default function Navigation() {
+  return (
+    <Nav>
+      <NavButton as={NavLink} exact to="/">
+        <HomeIcon className="gg-play-button-r" />
+        <TextWrap1>Play</TextWrap1>
+      </NavButton>
+      <NavButton as={NavLink} to="/movies">
+        <MoviesIcon className="gg-film" />
+        <TextWrap2>Movies</TextWrap2>
+      </NavButton>
+    </Nav>
+  )
+}
+
+const Nav = styled.nav`
+  position: fixed;
+  overflow-y: scroll;
+  bottom: 0;
+  display: flex;
+  gap: 10px;
+  background: #f8f8ff;
+  width: 100%;
+  padding: 20px;
+`
+
+const NavButton = styled(Button)`
+  font-size: 0.8em;
+  font-family: 'Courier New';
+  position: relative;
```

Comment:
Why are you using positioning here? Could it not be solved by grid or flexbox? The `TextWrapper1` and `TextWrapper2` components almost have the same style except for the positioning. Would be nicer to only have one `TextWrapper` component for both spans.

Reply:
Great idea, I've been struggling a lot with this one!! Thank you!!